src/mmw/apps/modeling/migrations/0036_no_overrides_by_default.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2021-07-28 17:30
+from __future__ import unicode_literals
+
+import django.contrib.postgres.fields.jsonb
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('modeling', '0035_project_layer_overrides'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='project',
+            name='layer_overrides',
+            field=django.contrib.postgres.fields.jsonb.JSONField(default=dict, help_text='JSON object of layers to override defaults with'),
```

Comment:
This ensures that the default is an empty dictionary, effectively no overrides, for all new projects.